satellite/satellitedb/users.go

```diff
@@ -78,6 +82,7 @@ func (users *users) Delete(ctx context.Context, id uuid.UUID) (err error) {
 // Update is a method for updating user entity
 func (users *users) Update(ctx context.Context, user *console.User) (err error) {
```

Comment:
We don't want to allow for the partner_id to be set after the account is created

Reply:
I think the diff here is misleading. The `Update` method is not changed. It's the `userFromDBX` function that's modified